src/Compilers/CSharp/Portable/Symbols/Source/SourceSimpleParameterSymbol.cs

```diff
@@ -82,6 +82,15 @@ internal override bool IsCallerMemberName
             get { return false; }
         }
 
+        internal override AttributeAnnotations FlowAnalysisAnnotations
+        {
+            get
+            {
+                (_, AttributeAnnotations annotations) = TryGetExtraAttributeAnnotations();
+                return annotations;
```

Comment:
>return annotations; [](start = 16, length = 19)

Do we have tests covering this code path?

Reply:
`NotNullWhenFalse_String_IsNullOrEmpty_WithoutCorlib` and `NotNullWhenFalse_String_IsNullOrWhiteSpace_WithoutCorlib` for case with annotations.
Most existing nullability tests for case with no annotations returned.